oidc-ui/configure_start.sh

```diff
@@ -18,7 +18,7 @@ unzip -o esignet-i18n-bundle.zip
 #sign-in-button-plugin
 echo "Downloading plugins"
 
-wget --no-check-certificate --no-cache --no-cookies $artifactory_url_env/artifactory/libs-release-local/esignet-plugins/sign-in-with-esignet.zip -O $plugins_path/temp/sign-in-button-plugin.zip
+wget --no-check-certificate --no-cache --no-cookies $SIGN_IN_WITH_ESIGNET_PLUGIN_URL -O $plugins_path/temp/sign-in-button-plugin.zip
```

Comment:
We will also need change in helm chart to pass the variable r8 @anshulv1401

Reply:
Yes, @ase-101. I will raise a DSD to update the same

Reply:
Raised a DSD for the updating of helm charts https://mosip.atlassian.net/browse/DSD-3416